File: 1374-generate-a-string-with-characters-that-have-odd-counts/1374-generate-a-string-with-characters-that-have-odd-counts.py
class Solution:
    def generateTheString(self, n: int) -> str:
        ans = ""
        if n % 2:
            for _ in range(n):
                ans += 'a'
        else:
            for _ in range(n-1):
                ans += 'a'
            ans += 'b'
        return ans

File: 1374-generate-a-string-with-characters-that-have-odd-counts/1374-generate-a-string-with-characters-that-have-odd-counts.cpp
class Solution {
public:
    string generateTheString(int n) {
        string result;
        if (n % 2 == 0) {
            for (int i = 0; i < n-1; i++) {
                result += 'a';
            }
            return result + 'b';
        } else {
            return string(n, 'a');
        }
    }
};